### PR TITLE
Rework nav to improve startup experience

### LINKED ISSUE #34428: Get Started user flow enhancements
**Suggestion to improve the Get Started user flow -**
1. "Get Started" links should point the user to a page with information regarding the creation of a new project,  such as the [Setup page](https://angular.io/guide/setup-local).
2. After that, we should provide Angular's architecture overview, something like [Architecture Overview](https://angular.io/guide/architecture) in simple terms.
4. Then, introduce the user to [Your First App](https://angular.io/start) as it is. Later ask the user to continue with the [Tour of Heroes App](https://angular.io/tutorial) tutorial to give them a real taste of developing an Angular app.

**Why these suggestions?**
* There is front-end framework war out in the world and every non-Angular developers say that documentations are hard to understand. This gives them one extra reason to move away from Angular.
* Few Front-end devs in my team informed me that they couldn't clearly get things which are required to start with Angular. They mentioned that they had to navigate a few pages to get started. When I tried a fresh start by myself, I too couldn't find things clearly, even though I have been visiting  [Angular.io](https://angular.io/) since 2017.

Above suggestions are from a few devs from my team(freshers and experienced). We will be more than happy to get further suggestions in order to improve the above starter flow even better.

### PR DESCRIPTION
**Pull request by jbogarthyde**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?
The startup experience is confusing

Issue Number: https://github.com/angular/angular/issues/34428
doc plan: https://angular-team.atlassian.net/browse/DOCS-815

## What is the new behavior?
Start -> setup
Add Start Up as a section
- emphasize concepts
- include Glossary
- distinguish between quick-start and tutorial
Move some "Fundamentals" to "Techniques"
Make doc-style guidelines visible, along w/code-style guidelines

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: aio/content/marketing/index.html [hunk 1]
```diff
@@ -104,10 +104,10 @@ <h1 class="no-anchor no-toc" style="display: none"></h1>
     <div layout="row" layout-xs="column" class="home-row">
       <a href="start">
         <div class="card">
-          <img src="generated/images/marketing/home/code-icon.svg" height="70px" alt="Getting Started with Angular">
+          <img src="generated/images/marketing/home/code-icon.svg" height="70px" alt="Get Started with Angular">
           <div class="card-text-container">
-            <div class="text-headline">Get Started</div>
-            <p>Start building an Angular application.</p>
+            <div class="text-headline">Try it now</div>
+            <p>Explore Angular's capabilities with our ready-made sample app. No setup required.</p>
```
Comment (kapunahelewong) [inline aio/content/marketing/index.html]:
```suggestion
            <p>Explore Angular's capabilities with a ready-made sample app. No setup required.</p>
```

**Review by kapunahelewong — APPROVED** [review]
Just a wee thing. :)

**Comment by jbogarthyde** [comment]
@petebacondarwin @gkalpak 
This is failing a test because it wants "start" to point to the StackBlitz "Get Started" tutorial, which is in the `aio/content/start` dir. But we want to change most of those pointers (which appear in several places) to point to the "Local Setup" page instead (`content/guide/setup-local`). 

This PR changes the nav so that both tutorials appear under a new "Getting Started" section, and are more clearly distinguished from each other in their intended audience.

I'm not sure which link is failing - I think it's the "Get Started" button on the Angular landing page.

Can we change the test?

 https://circleci.com/gh/angular/angular/632771?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

File: aio/content/guide/bootstrapping.md [hunk 2]
```diff
@@ -7,12 +7,11 @@ A basic understanding of the following:
 
 <hr />
 
-An `NgModule` describes how the application parts fit together.
-Every application has at least one Angular module, the _root_ module
-that you bootstrap to launch the application.
-By convention, it is usually called `AppModule`.
+Every application has at least one Angular module, the _root_ module,
+which must be present for bootstrapping the application on launch.
+By convention and by default, this NgModule is named `AppModule`.
```
Comment (aikidave) [inline aio/content/guide/bootstrapping.md]: We should define NgModule. I think that got dropped with the rewrite.

File: aio/content/guide/bootstrapping.md [hunk 3]
```diff
@@ -7,12 +7,11 @@ A basic understanding of the following:
...
 
-If you use the [Angular CLI](cli) to generate an app, the default `AppModule` is as follows:
+When you use the [Angular CLI](cli) command `ng new` to generate an app, the default `AppModule` is as follows.
```
Comment (aikidave) [inline aio/content/guide/bootstrapping.md]: I would offset the command with commas: command, `ng new`, to generate...

I am not sure why I feel that way.

File: aio/content/guide/observables.md [hunk 4]
```diff
@@ -1,5 +1,5 @@
 
-# Observables
+# Introduction to observables
 
 Observables provide support for passing messages between publishers and subscribers in your application. Observables offer significant benefits over other techniques for event handling, asynchronous programming, and handling multiple values.
```
Comment (aikidave) [inline aio/content/guide/observables.md]: Should we change the title to: "Passing messages?"

Also, the terms publishers and subscribers may be unfamiliar to new developers?

File: aio/content/guide/router.md [hunk 5]
```diff
@@ -1,7 +1,8 @@
-# Routing and navigation
+# In-app navigation: routing to views
```
Comment (aikidave) [inline aio/content/guide/router.md]: I like this! Covers both the user task and the feature

File: aio/content/guide/schematics.md [hunk 6]
```diff
@@ -1,4 +1,4 @@
-# Schematics
+# Schematics overview
 
 A schematic is a template-based code generator that supports complex logic.
```
Comment (aikidave) [inline aio/content/guide/schematics.md]: Can we make this more user-focused?

# Creating code templates

Or something similar?

File: aio/content/guide/web-worker.md [hunk 7]
```diff
@@ -1,17 +1,26 @@
-# Using web workers with Angular CLI
+# Using web workers for background processing
```
Comment (aikidave) [inline aio/content/guide/web-worker.md]: I like this, but maybe reverse it to put emphasis on the user task?

# Background processing using web workers

File: aio/content/guide/web-worker.md [hunk 8]
```diff
@@ -36,11 +45,10 @@ Running this command will:
   }
   </code-example>
 
-After the initial scaffolding, you will need to refactor your code to use the web worker by sending messages to and from.
+After the initial scaffolding, you will need to refactor your code to use the web worker by sending messages to and from the worker.
```
Comment (aikidave) [inline aio/content/guide/web-worker.md]: Is scaffolding a term that the audience understands?

Comment (aikidave) [inline aio/content/guide/web-worker.md]: Also, perhaps change "will need to" to "must" (shorter)

File: aio/content/guide/web-worker.md [hunk 9]
```diff
@@ -36,11 +45,10 @@ Running this command will:
...
 
-## Caveats
+<div class="alert is-important>
 
-There are two important things to keep in mind when using web workers in Angular projects:
+Some environments or platforms, such as `@angular/platform-server` used in [Server-side Rendering](guide/universal), don't support web workers. You must provide a fallback mechanism to perform the computations that the worker would otherwise perform, in order to ensure that your application will work in these environments.
```
Comment (aikidave) [inline aio/content/guide/web-worker.md]: Can we link them to a page that describes how to provide a fallback mechanism?

**Review by aikithoughts — COMMENTED** [review]
Some initial thoughts! I like where this is going.

File: aio/content/guide/bootstrapping.md (hunk 3 above)
Comment (jbogarthyde) [reply aio/content/guide/bootstrapping.md]: I don't agree, I think it looks fussy.

File: aio/content/guide/web-worker.md (hunk 9 above)
Comment (jbogarthyde) [reply aio/content/guide/web-worker.md]: I don't know of such a page. There used to be a whole web-workers package, but it was deprecated and removed. Would need an engineer to provide an example.

File: aio/content/guide/bootstrapping.md (hunk 3 above)
Comment (aikidave) [reply aio/content/guide/bootstrapping.md]: Fair enough!

**Comment by aikidave** [comment]
I really appreciate all the work you've put into this! I also, though, think it's best that we split this work into multiple PRs, as there are several changes here that we should track separately. My recommendations:

- Remove the following files into into their own PRs. There should be one PR for each file:
  - bootstrap.md
  - http.md
  - observables.md
  - providers.md
  - router.md (Note: this one may be obsolete due to PR #35566 )
  - schematics.md
  - web-worker.md
**Reason**: These changes do not seem directly tied to our effort to improve the getting started experience.
- Move issues in which we replace `page` with `view` to a separate PR.
  **Reason**: This seems to be a terminology change, and we should consult with subject matter experts to ensure it is the right term to use.

There are several ways to separate out these files. Here's one option:

https://stackoverflow.com/questions/30768148/git-split-pull-request-into-smaller-prs-based-upon-the-new-directories-in-the

The Angular community likely has other options that may work as well.

**Comment by jbogarthyde** [comment]
Moved non-strictly-startup changes into individual PRs:

https://github.com/angular/angular/pull/36021
https://github.com/angular/angular/pull/36023
https://github.com/angular/angular/pull/36024
https://github.com/angular/angular/pull/36026

These are mostly title changes to implement a task-oriented approach.
I did not remove any of the changes from this PR -- and after those PRs land, there may still be some conflicts/amendments from this PR, probably relating to nav links and descriptions.

**Comment by kapunahelewong** [comment]
Please merge https://github.com/angular/angular/pull/36281 before this PR merges. Blocking as a reminder.

**Review by aikithoughts — APPROVED** [review]

File: aio/content/marketing/index.html [hunk 10]
```diff
@@ -15,7 +15,7 @@
     <!-- CONTAINER -->
     <div class="homepage-container">
       <div class="hero-headline no-toc">One framework.<br>Mobile &amp; desktop.</div>
-      <a class="button hero-cta" href="start">Get Started</a>
+      <a class="button hero-cta" href="guide/setup-local">Get Started</a>
```
Comment (IgorMinar) [inline aio/content/marketing/index.html:18]: Have we thought this particular change through? If we are going to make guide/setup-local the entry point into the getting started journey then we should consider redirecting angular.io/start to guide/setup-local as well.

I suspect that the link "angular.io/start" is currently published all over the place so just changing the url in this link is likely not sufficient to get the desired final effect.

Comment (IgorMinar) [reply aio/content/marketing/index.html:18]: After more digging, I'm now wondering if this link shouldn't point to angular.io/docs instead. That seems much more appropriate (especially if we make /docs page a big more visually appealing). my 2c.

**Review by IgorMinar — APPROVED** [review]
I love this simplified user journey! Great work!

Can we get a clarification where the "Getting started" link on the front page of angular.io should point to?

**Review by atscott — APPROVED** [review]
reviewed-for: fw-router

File: aio/content/guide/docs-style-guide.md [hunk 11]
```diff
@@ -1010,21 +1010,21 @@ Callouts use the same _urgency levels_ that alerts do.
 <div class="callout is-critical">
 <header>A critical point</header>
 
-**Pitchfork hoodie semiotics**, roof party pop-up _paleo_ messenger messenger bag cred Carles tousled Truffaut yr. Semiotics viral freegan VHS, Shoreditch disrupt McSweeney's. Intelligentsia kale chips Vice four dollar toast, Schlitz crucifix
+**Pitchfork hoodie semiotics**, roof party pop-up _paleo_ messenger messenger bag cred Carles tousled Truffaut yr. Semiotics viral freegan VHS, Shoreditch disrupt McSweeney's. Intelligentsia kale chips Vice four dollar toast
```
Comment (alxhub) [inline aio/content/guide/docs-style-guide.md:1013]: What is this, techy lorum ipsum? :D

Comment (jbogarthyde) [reply aio/content/guide/docs-style-guide.md:1013]: yes, exactly. Historical - before my time, but we just wanted to remove potentially objectionable bits before surfacing this page in the nav. (We are working on updating this page in a separate PR.)

**Review by alxhub — APPROVED** [review]

File: aio/content/marketing/index.html (hunk 10 above)
Comment (IgorMinar) [reply aio/content/marketing/index.html]: @jbogarthyde can you please clarify how was this concern resolved? I see your thumbs up which usually means that you are accepting my suggestion, but in this case I don't see the corresponding update in this PR. Thank you!

Comment (IgorMinar) [reply aio/content/marketing/index.html]: oh.. nevermind. I do see now that you changed it to "docs".. great! thank you!

**Comment by jbogarthyde** [comment]
I made the “Get Started” button on the home page go to the Docs home page, as you suggested.

**Pull request closed without merging**